proceeding/tables/catalog_table.tex

```diff
@@ -0,0 +1,21 @@
+% Table
+\begin{table}[h]
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || lrll ||}
+\hline
+% \tch{1}{c}{b}{Catalog\\ name} & \tch{1}{c}{b}{Number\\ of sources}  & \tch{1}{c}{b}{Update}  & \tch{1}{c|}{b}{Description}  \\
+% & \tch{1}{c}{b}{Single\\ outlet}  & \tch{1}{c}{b}{Small\\ multiple\tabnoteref{t1n1}}  & \tch{1}{c}{b}{Large\\ multiple}  & \tch{1}{c}{b}{Total}   \\
+
+
+\textbf{Catalog} & \textbf{Sources} & \textbf{Update} & \textbf{Description} \\ \hline
+\textbf{3FGL} & 3034 & fixed & Third Fermi-LAT point source catalog\\
+\textbf{2FHL} & 360 & fixed & Second Fermi-LAT Catalog of High-Energy Sources\\
+\textbf{SNRcat} & 378 & continuous & A census of high-energy observations of Galactic supernova remnants\\
+\textbf{TeV} & 155 & continuous & The TeGeV Catalogue @ ASDC\\
```

Comment:
Maybe put in this order?
TeV, 2FHL, 3FGL, SNRcat

Or at least group the gamma-ray catalogs together, and put the MWL catalogs (only SNRCat at the mometn) below.